edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.Core.Test/ConnectionManagerTest.cs

```diff
@@ -132,7 +132,7 @@ public async Task CloudConnectionTest()
             await connectionManager.RemoveDeviceConnection(deviceCredentials2.Identity.Id);
 
             returnedValue = await connectionManager.GetCloudConnection(deviceCredentials2.Identity.Id);
-            Assert.True(returnedValue.HasValue);
+            Assert.False(returnedValue.HasValue);
```

Comment:
Should `deviceCredentials1` be removed at the end of the test as well?

Reply:
agreed that this is nice to clean up, but this is breaking anything and clean up may need for other tests in this test class as well.  It should be done in a separate PR and in master branch only.

Reply:
Add more assert statement for deviceCredentials1 cloud proxy.